src/content/docs/apm/transactions/key-transactions/introduction-key-transactions.mdx

```diff
@@ -16,33 +16,230 @@ redirects:
+
+</Collapser>
+
+<Collapser
+    id="create-service-levels"
+    title="Create service levels for key transactions."
+  >
+In addition to alert thresholds, you can also measure the performance from your application users' point of view. To do this, set up [service levels](/docs/service-level-management/intro-slm/) against the business key transactions.
+
+</Collapser>
+  </CollapserGroup>
+
+## Use key transactions in context 
+
+After you've created a key transaction, added tags, created workloads, and set up alerts, you can start to analyze your data in context with other entities that you monitor. Once you understand its role within your wider system you can take any additional steps to improve uptime and reliability.  
+
+<CollapserGroup>
+<Collapser
+    id="lookout"
+    title="Analyze deviations with New Relic Lookout"
+  >
+
+To quickly analyze deviations in all your key transactions, you can start with New Relic [Lookout](/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-lookout-monitor-your-estate-glance/). This provides a real-time view of your transactions across all your accounts, highlighting changes in all of your telemetry. In this example, you can see the `getPlans` transaction shows an increase in response time:
+
+<img
+  title="New Relic Lookout and key transactions"
+  alt="A screenshot depicting New Relic Lookout and key transactions."
+  src={apmLookoutandKeyTransactions}
+/>
+</Collapser>
+
+
+<Collapser
+    id="service-maps"
+    title="Understand impact through service maps"
+  >
+
+To understand the complete impact of the `getPlans` transaction performance degradation, you can use a [service map](/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/introduction-service-maps/). You can visualize the health of all the other connected entities like the originating service, hosts, databases, and other key transactions.
+
+<img
+  title="Service maps and key transactions"
+  alt="A screenshot depicting service level maps and key transactions"
+  src={apmServiceMapsandKeyTransactions}
+/>
+
+</Collapser>
+  <Collapser
+    id="change tracking"
+    title="Track changes"
+  >
+
+To track the impact of any change event, such as code deployments, configuration changes, and business events, you can use the [change tracking markers](/docs/change-tracking/change-tracking-view-analyze/) on the key transaction charts. This helps you drill down further in the trace to troubleshoot the changes that are causing issues. 
```

Comment:
@homelessbirds I'm pretty familiar with the change tracing routine, and it seems you would first need to designate key transactions to track using GraphQL, the CLI, or some kind of CI/CD tool like Jenkins. I'm not sure we cover the the key transactions use case in our docs (I could be mistaken).

Reply:
good catch, thanks!